Content.Server/Zombies/ZombieSystem.cs

```diff
@@ -8,6 +8,7 @@
 using Content.Server.Inventory;
 using Content.Server.Speech.EntitySystems;
 using Content.Shared.Bed.Sleep;
+using Content.Shared.Cloning;
```

Comment:
Is it intended?

Reply:
Yeah since zombies use the cloning system.